.woodpecker.yml

```diff
@@ -27,6 +27,11 @@ pipeline:
     commands:
       - prettier -c . '!**/volumes' '!**/dist' '!target' '!**/translations'
 
+  taplo_check:
+    image: tamasfe/taplo
```

Comment:
This should use a specific version tag, not latest.

Reply:
Done, although with a lot of these lint-type things, I'd almost prefer they use latest. Then we can quickly see that builds are breaking, and get on it fast, while still supporting the newest preferred formats.

Reply:
That means all CI builds including main branch and pull requests will break. With a fixed version, it will only break in pull requests from developers who have updated their local formatter. Thats much easier to handle.